keyboards/cipulot/kawayo/chconf.h

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2022 Cipulot
```

Comment:
This file should be able to be deleted

Reply:
I can confirm. Removing this file doesn't affect workings on the firmware with the hardware.